### PR TITLE
Kaggle benchmark 2

### PR DESCRIPTION
**Pull request by aptsunny**
*Issue #, if available:*

*Description of changes:*
kaggle classification benchmark

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: examples/image_classification/benchmark.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Documents for Kaggle Benchmark
+
+## How to build
```
Comment (zhanghang1989) [inline examples/image_classification/benchmark.md]: remove installation

File: examples/image_classification/benchmark.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+1. `pip uninstall autogluon`
+2. `python setup.py develop`
+
+## Prepare the dataset
+
+1. Assume your instance can visit S3.
+
+2. `aws s3 cp s3://zhiz-data/datasets/a.zip /your_path/`
```
Comment (zhanghang1989) [inline examples/image_classification/benchmark.md]: Use Instruction instead.

File: examples/image_classification/benchmark.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+3. `unzip a.zip -d dataset`
+
+## Train the dataset with fit function:
+
+1. ```dataset = 'dogs-vs-cats-redux-kernels-edition/'```
```
Comment (zhanghang1989) [inline examples/image_classification/benchmark.md]: use argparser.

Comment (zhanghang1989) [reply examples/image_classification/benchmark.md]: python autogluon/examples/image_classification/benchmark.py --dataset dogs

File: autogluon/task/image_classification/dataset.py [hunk 4]
```diff
@@ -4,10 +4,12 @@
 import logging
 import numpy as np
 from PIL import Image
-
 import mxnet as mx
 from mxnet import gluon, nd
 from mxnet.gluon.data import Dataset as MXDataset
+
+# from mxnet.gluon.data.vision import ImageRecordDataset, transforms
+
```
Comment (chongruo) [inline autogluon/task/image_classification/dataset.py]: remove commented-out code

File: autogluon/task/image_classification/dataset.py [hunk 5]
```diff
@@ -35,6 +36,10 @@ def get_dataset(path=None, train=True, name=None,
     """ Method to produce image classification dataset for AutoGluon, can either be a 
     :class:`ImageFolderDataset`, :class:`RecordioDataset`, or a 
     popular dataset already built into AutoGluon ('mnist', 'cifar10', 'cifar100', 'imagenet').
+<<<<<<< HEAD
+
+=======
+>>>>>>> origin/kaggle_benchmark
```
Comment (chongruo) [inline autogluon/task/image_classification/dataset.py]: remove conflict marker

File: autogluon/task/image_classification/classifier.py [hunk 6]
```diff
@@ -15,6 +15,9 @@
 from ...utils import save, load, tqdm
 from ...utils.pil_transforms import *
 
+# from collections import OrderedDict
+# from mxnet.gluon.block import collect_params
```
Comment (chongruo) [inline autogluon/task/image_classification/classifier.py]: remove commented-out code

File: autogluon/task/image_classification/classifier.py [hunk 7]
```diff
@@ -85,26 +88,32 @@ def predict(self, X, input_size=224, plot=True):
         def predict_img(img):
             # load and display the image
             proba = self.predict_proba(img)
+            # extract probability
```
Comment (chongruo) [inline autogluon/task/image_classification/classifier.py]: same here

File: docs_bak/LICENSE [hunk 8]
```diff
@@ -0,0 +1,438 @@
+Attribution-NonCommercial-ShareAlike 4.0 International
```
Comment (zhanghang1989) [inline docs_bak/LICENSE]: What is this file?

File: docs_bak/README.md [hunk 9]
```diff
@@ -0,0 +1,7 @@
+# Documents for AutoGluon
```
Comment (zhanghang1989) [inline docs_bak/README.md]: Why duplicating files?

**Pull request closed without merging**